azerothcore/azerothcore-wotlk
# feat(Core/Logs): Improve logs for linked_respawn

## Changes Proposed:
-  Give actual information about the error instead a vague message. (From trinity)

## Issues Addressed:

- None

## SOURCE:


## Tests Performed:

- Built, not tested.


## How to Test the Changes:


1. Delete any creature which has a linked_respawn.
2. See the startup errors, now it should give you more information about how to fix.
